src/main/webapp/enroll.html

```diff
@@ -81,34 +85,27 @@ <h3 class="panel-title">Version Being Released</h3>
                 <div class="panel-body">
                     <div class="form-group">
                         <label class="control-label">Version Being Released</label>
-                        <input maxlength="200" type="text" required="required" class="form-control" placeholder="Enter Version being Released"/>
+                        <input maxlength="200" type="text" required="required" class="form-control" placeholder="Enter Version being Released" name="versionReleased"/>
```

Comment:
nit: The name sounds to indicate that it is an already released version. Change to newVersion or newReleaseVersion?